projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -74,6 +74,19 @@ def __init__(self, config_file=None):
             self.get_value("RUN_TEST_TIMEOUT_SECONDS", "1200")
         )
 
+        # Ingestion configuration
+        self.INGEST_TARGET = self.get_value(
+            "INGEST_TARGET",
+            "vdk-heartbeat-datasource",
+        )
+
+        self.INGEST_METHOD = self.get_value("INGEST_METHOD", "http")
+
+        self.INGEST_DESTINATION_TABLE = self.get_value(
+            "INGEST_DESTINATION_TABLE",
```

Comment:
Let's prefix them with VDK_HEARTBEAT ?

Reply:
Sounds meaningful. I aligned them with the rest of the variables.